stdeb/util.py

```diff
@@ -1248,6 +1252,7 @@ def build_dsc(debinfo,
               debian_dir_only=False,
               sign_dsc=False,
               ignore_source_changes=False,
+              compat=9
```

Comment:
Shouldn't the code calling this function be updated to pass the configured compat version? Otherwise this would be a hard coded value again.